Projects/Contracts/Sources/View/TerminationSuccessScreen.swift

```diff
@@ -29,9 +29,30 @@ public struct TerminationSuccessScreen: View {
         .padding(.bottom, -100)
 
         hButton.LargeButtonFilled {
+
+            let currentMarket = Localization.Locale.currentLocale.market
+            var surveyURL = ""
+
+            switch currentMarket {
+            case .se:
+                surveyURL =
+                    "https://hedvigapp.typeform.com/to/YHVVx1#memberid=xxxxx&contract_id=xxxxx&contract_name=xxxxx&terminated_contracts_count=xxxxx"
+            case .no:
+                surveyURL = "https://hedvigapp.typeform.com/to/a7aRZzir#memberid=xxxxx"
+            case .dk:
+                surveyURL = "https://hedvigapp.typeform.com/to/dk9Dj7S8#memberid=xxxxx"
+            default:
```

Comment:
Avoid putting defaults, because if we add a new case the code wills till compile, but this functionality is unhandled.

Reply:
Should I change default: to                 

case .fr:
 break

? Because then I need to include that option :)

Reply:
Ah, time to remove `fr` i guess, you could do a `fatalError("not supported in FR")` for that case maybe?

Reply:
Added :)